pkg/process/util/netns.go

```diff
@@ -120,5 +131,5 @@ func GetInoForNs(ns netns.NsHandle) (uint64, error) {
 		return 0, err
 	}
 
-	return s.Ino, nil
+	return uint32(s.Ino), nil
```

Comment:
An inode number is a 64 bits.
Is it safe to truncate it ?

Reply:
The number we are interested in, the network namespace number, is an `unsigned int`, which is 32bits. 
https://github.com/torvalds/linux/blob/f69d02e37a85645aa90d18cacfff36dba370f797/include/linux/ns_common.h#L12

Reply:
Perfect ! Thanks for the explanation !